shared/actions/config/index.tsx

```diff
@@ -332,7 +332,7 @@ const startLogoutHandshake = (state: Container.TypedState) =>
 // stuff to trigger this due to a timeout if there's no listeners or something
 function* maybeDoneWithLogoutHandshake(state: Container.TypedState) {
   if (state.config.logoutHandshakeWaiters.size <= 0) {
-    yield RPCTypes.loginLogoutRpcPromise({force: false})
+    yield RPCTypes.loginLogoutRpcPromise({force: false, keepSecrets: false})
```

Comment:
when does this get called?

Reply:
When you click "log out" in the GUI and the GUI determines that you're safe to log out